sdk/azurestack/Microsoft.AzureStack.Management.Storage.Admin/tests/Microsoft.AzureStack.Management.Storage.Admin.Tests.csproj

```diff
@@ -7,7 +7,7 @@
     <VersionPrefix>1.0.0</VersionPrefix>    
   </PropertyGroup>
   <!--<PropertyGroup>
```

Comment:
Should we remove these commented out properties instead of updating them?

Reply:
Yep, I'll clean up mechanical find-replace leftovers